### ISSUE TITLE
BYOL projection weights

### ISSUE THREAD

**Issue by alex-xun-xu** [deepmind/deepmind-research#256]
I am very interested in trying to adapt the pre-trained model to additional pre-training on other datasets. But I found the weights for projection head missing. So can you please share the BYOL projection weights for ResNet50 as well?

**Comment by altche** [comment]
Hi, thanks for reaching out.

The full experiment state (including parameters and BatchNorm statistics for the ResNet, projector, predictor and classifier) are in the downloadable checkpoints in the README.

Specifically for your question, the weights and statistics for the projection/prediction heads are in the following dictionaries:
```
checkpoint['experiment_state'].(online|target)_(params|state)
```
I think you are interested in the following keys in `(online|target)_params`:
```'predictor/batch_norm', 'predictor/linear', 'predictor/linear_1', 'projector/batch_norm', 'projector/linear', 'projector/linear_1'``` where `*/linear` is the first linear layer and `*/linear_1` the second in application order. You can find similar keys in `(online|target)_state` for the bachnorm stats.

Best,
Florent
